CMakeLists.txt

```diff
@@ -85,8 +85,8 @@ project_options(
   ${ENABLE_COVERAGE}
   ${ENABLE_SANITIZER_ADDRESS}
   ${ENABLE_SANITIZER_UNDEFINED_BEHAVIOR}
-  # ${ENABLE_SANITIZER_THREAD}
-  # ${ENABLE_SANITIZER_MEMORY}
+  ${ENABLE_SANITIZER_THREAD}
+  ${ENABLE_SANITIZER_MEMORY}
```

Comment:
These sanitizers cannot be enabled together

Reply:
done